app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -106,6 +106,10 @@ def account(form526_submission)
       Account.find_by(edipi: edipi) if edipi
     end
 
+    def patient_info(form526_submission)
+      form526_submission.full_name.merge(birthdate: form526_submission.auth_headers['va_eauth_birthdate'])
+    end
```

Comment:
Pulled this over unchanged from the hypertension job, since it's equally relevant to asthma.

Reply:
This may be something that could just go into the new RRD concern for the model but makes sense here

Reply:
Acknowledged. It can go into `RrdProcessor` https://github.com/department-of-veterans-affairs/vets-api/pull/9577/files#diff-10ef8ac3ff61abaac78e7986500c46a625c44c551a2eeee043939c72ad2408a0R4.
I still need to split the PR further into smaller PRs.